packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -55,7 +55,7 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents, TaikoErrors {
         LibVerifying.init(state, getConfig(), _genesisBlockHash);
     }
 
-    function init2() external reinitializer(2) {
+    function init2() external onlyOwner reinitializer(2) {
```

Comment:
As effectively this `reinitializer` is fully harmless, and that `onlyOwner` might be `Timelock` (?), i'm not sure if we need to guard `onlyOwner`, what do you think ?

Reply:
In this case, onlyOwner is not really necessary, but I think maybe it's a good practice to ensure reinitializers are only called by owners so the owner controls the timing, which is probably important in case the protocol needs to be upgraded first, then the node is updated, the a feature is activated, then the reinitializer shall be called.